### PR TITLE
Update building.md for windows

### PR DESCRIPTION
**Pull request by coco875**
update building.md

### CODE REVIEW

File: doc/building.md [hunk 1]
```diff
@@ -55,6 +55,17 @@ cmake ..
 make -j$(nproc)
 ```
 
+_Note:_ on windows is
+```powershell
+cd nyan
+mkdir build
+cd build
+cmake .. -G "Visual Studio 16 2019"
```
Comment (TheJJ) [inline doc/building.md:63]: the generator depends on your ide target, and i wouldn't hardcode a 4 year old version here. isnt't there a better flag?

Comment (coco875) [reply doc/building.md:63]: I don't try to remove the flag to see if it still work or not

Comment (coco875) [reply doc/building.md]: I work without it

File: doc/building.md [hunk 2]
```diff
@@ -55,6 +55,17 @@ cmake ..
 make -j$(nproc)
 ```
 
+_Note:_ on windows is
+```powershell
+cd nyan
+mkdir build
+cd build
+cmake ..
+cmake --build . --config RelWithDebInfo
```
Comment (TheJJ) [inline doc/building.md:64]: ok, now the commands are basically identical apart from this line -> we can just unify and say "if you used makefiles as generator target (linux, mac, ...): `make ...`
if you are on windows or want to let cmake start the build: `cmake --build ...`"

then its not redundant but still acurate :)

Comment (coco875) [reply doc/building.md:60]: ok fix

**Review by TheJJ — APPROVED** [review]

**Pull request merged**